### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by necessarily-equal**
This PR needs https://github.com/DaemonEngine/Daemon/pull/516 to be merged first, and submodules synced.

### CODE REVIEW

File: src/sgame/sg_cm_world.cpp [hunk 1]
```diff
@@ -113,14 +113,8 @@ clipHandle_t G_CM_ClipHandleForEntity( const gentity_t *ent )
 		return CM_InlineModel( ent->s.modelindex );
 	}
 
-	if ( ent->r.svFlags & SVF_CAPSULE )
-	{
-		// create a temp capsule from bounding box sizes
-		return CM_TempBoxModel( ent->r.mins, ent->r.maxs, true );
-	}
-
 	// create a temp tree from bounding box sizes
```
Comment (slipher) [inline src/sgame/sg_cm_world.cpp:116]: I don't understand this comment but maybe it's better to get rid of it since it was on only one of the cases?

Comment (necessarily-equal) [reply src/sgame/sg_cm_world.cpp]: I've updated the comment to say "tree/capsule"

**Comment by slipher** [comment]
Looks good, just waiting on the submodule thing.

**Comment by slipher** [comment]
Hmm actually it's the other way around, this needs to be merged before Daemon#516. Apparently we did not need to wait on Daemon.

**Pull request merged**